### PR TITLE
Remove simplemobiletools from dependencies

### PR DESCRIPTION
**Pull request by shubertm**

### CODE REVIEW

**Comment by tuancoltech** [comment]
@shubertm Have you tried building on this branch including changes from https://github.com/ARK-Builders/ARK-Memo/pull/69? Build is still successful?

File: app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferences.kt [hunk 1]
```diff
@@ -8,4 +8,8 @@ interface MemoPreferences {
     fun getPath(): String
 
     fun getNotesStorage(): Path
+
+    fun storeCrashReportEnabled(bool: Boolean)
```
Comment (tuancoltech) [inline app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferences.kt:12]: nit: Could we have a more meaningful name for the param? `bool` is not the best one, I guess.

File: app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferencesImpl.kt [hunk 2]
```diff
@@ -26,4 +27,13 @@ class MemoPreferencesImpl @Inject constructor(@ApplicationContext context: Conte
     override fun getPath(): String = sharedPreferences.getString(CURRENT_NOTES_PATH, "") ?: ""
 
     override fun getNotesStorage(): Path = Path(getPath())
+
+    override fun storeCrashReportEnabled(bool: Boolean) {
+        prefEditor.apply {
+            putBoolean(CRASH_REPORT_ENABLE, bool)
+            apply()
+        }
```
Comment (tuancoltech) [inline app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferencesImpl.kt:35]: Could this be simplified to one line of code as below?

```suggestion
prefEditor.putBoolean(CRASH_REPORT_ENABLE, bool).apply()
```

**Comment by shubertm** [comment]
> @shubertm Have you tried building on this branch including changes from https://github.com/ARK-Builders/ARK-Memo/pull/69? Build is still successful?

No, we should try

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -101,5 +101,6 @@
     <string name="toast_save_qr_success">QR saved at %s</string>
     <string name="tips_will_be_available_soon">Will be available soon!</string>
     <string name="toast_invalid_recording">Invalid recording file</string>
+    <string name="ark_memo_play">Play</string>
```
Comment (tuancoltech) [inline app/src/main/res/values/strings.xml:104]: The original string is `Play / Pause`. Let's keep it the same.

File: app/src/main/res/layout/dialog_donate_qr.xml [hunk 4]
```diff
@@ -71,7 +71,7 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintEnd_toStartOf="@+id/layout_copy"
-            android:layout_marginVertical="@dimen/medium_margin"
+            android:layout_marginVertical="12dp"
```
Comment (tuancoltech) [inline app/src/main/res/layout/dialog_donate_qr.xml:74]: I believe the original dimension is `8dp`, unless we explicitly want to change the UI.

**Comment by tuancoltech** [comment]
> > @shubertm Have you tried building on this branch including changes from #69? Build is still successful?
> 
> No, we should try

@shubertm So now after merging the workflow update, is the build successful in your machine?

**Comment by shubertm** [comment]
> > > @shubertm Have you tried building on this branch including changes from #69? Build is still successful?
> > 
> > No, we should try
> 
> @shubertm So now after merging the workflow update, is the build successful in your machine?

Yes, it's successful

**Comment by shubertm** [comment]
The comments have been resolved. Please verify again @tuancoltech

**Pull request merged**